rules/web/webserver_generic/web_sql_injection_in_access_logs.yml

```
@@ -41,6 +41,9 @@ detection:
         - 'select%20*%20'
         - 'or 1=1#'
         - 'or%201=1#'
+        - 'concat_ws('
+        - 'select database()'
+        - 'select version()'
```

```suggestion
        - 'select database()'
        - 'select version()'
        - 'select%20database()'
        - 'select%20version()'
```
to remain constant with the rest of the rule

---

Thanks @frack113! Good additions